components/raftstore/src/store/fsm/apply.rs

```diff
@@ -2283,6 +2286,7 @@ impl Apply {
         commit_term: u64,
         commit_index: u64,
     ) -> Apply {
+        APPLY_PENDING_BYTES_GAUGE.add((ENTRY_MEM_SIZE * entries.capacity()) as i64);
```

Comment:
This is not correct. Entries data size should be recorded too.

Reply:
Good catch. I will fix it tonight.

Reply:
Fixed.